### PR TITLE
Create CONTRIBUTING

### PR DESCRIPTION
**Pull request by celbal**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* All modified code is required to pass our [Coding Standards and Style Guide](https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html).
+* You must write or update tests to exercise any modified code.
+* You must provide test data for any new or modified tests.
+* You must have a Redmine issue assigned to you before submitting a pull request.
+* You must complete the Developer Checklist before submitting your pull request.
+* When submitting a pull request, you must include the Redmine issue number in the title of the pull request. Use the "Fixes #0000" format.
+
+## What can I expect from the ISIS3 development team?
+### Response Time
+Our response time will depend on availablity and scheduling.
```
Comment (ihumphrey) [inline CONTRIBUTING.md]: forgot an "i" in availability

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+### Feedback
+Our feedback will address any discrepencies related to the contribution guidelines, our coding standards and style guide, and our code review checklist.
```
Comment (ihumphrey) [inline CONTRIBUTING.md]: discrepancies

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+If you have a question or a problem with contributing to our software please contact ## REGENT EMAIL HERE ##.
+
+## How can I contribute?
+
+### Redmine Issues
+If you have a question about, a problem with, or a suggestion for ISIS3, please read our [Guidelines for reporting Redmine issues](https://isis.astrogeology.usgs.gov/fixit/projects/isis/wiki/Guidelines_for_Reporting_Issues). Once you have read that please [submit](https://isis.astrogeology.usgs.gov/fixit/projects/isis/issues/new) a Redmine issue.
+
+### Working on an existing issue or enhancement
+1. Assign the redmine issue to yourself. If you don't have the permission to do this, please make a note on the issue that you would like to work on it. Please contact ## REGENT EMAIL HERE ## for permission to work on an issue.
```
Comment (ihumphrey) [inline CONTRIBUTING.md]: I don't know which is correct, but it might be good to be consistent with Redmine/redmine capitalization

**Review by ihumphrey — DISMISSED** [review]
Looks like a good start to our contributions page

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+[What can I expect from the ISIS3 development team](#What-can-I-expect-from-the-ISIS3-development-team)
+  * [Response time](#Response-time)
+  * [Feedback](#feedback)
+
+[Development Process Checklists](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developer-Checklists)
+
+[References](#references)
+
+## I have a question or a problem!
+If you have a question about or a problem with ISIS3, please see the [Redmine Issues](#redmine-issues) section.
```
Comment (krlberry) [inline CONTRIBUTING.md:23]: Suggestion during the meeting today: put a direct link to Redmine somewhere in here and also explain what Redmine is.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+3. Work with an ISIS3 developer, the issue reporter, and any other interested parties to get feedback on your solution. This may be an iterative process.
+4. Add an [impact statement](https://isis.astrogeology.usgs.gov/fixit/projects/isis/wiki/Impact_Statement) to the redmine issue. If you don't have the permission to do this, please make a note on the issue with the impact statement.
+5. Make the changes on your fork of the ISIS3 main GitHub repo. Follow the [Code Base Contribution Guidelines](#code-base-contribution-guidelines).
+6. Follow the Developer's Checklist to ensure your changes are ready for review.
+7. Make a pull request. Include the redmine issue number in the title of the pull request. Use the "Fixes #0000" format.
+8. Work with your code reviewer, tester, and reporter to improve your changes. Your pull request will not be merged in until all parties approve the changes.
+9. Check in your test data. If you are an outside contributer, work with your code reviewer to ensure your tests and test data get checked in.
+
+### Working on a new issue or enhancement
+1. Please create a [Redmine issue](#redmine-issues).
```
Comment (krlberry) [inline CONTRIBUTING.md]: Suggestion (from Scott?) from during the meeting today: Move this to the above section with something along the lines of "if this issue does not exist, create one."

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+* Personal style changes will not be accepted.
+* Changes to bring code closer to our [Coding Standards and Style Guide](https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html) are encouraged.
+* Please be professional, even in comments.
+* Variable names must be meaningful.
+* All modified code is required to pass our [Coding Standards and Style Guide](https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html).
+* You must write or update tests to exercise any modified code.
+* You must provide test data for any new or modified tests.
+* You must have a Redmine issue assigned to you before submitting a pull request.
+* You must complete the Developer Checklist before submitting your pull request.
+* When submitting a pull request, you must include the Redmine issue number in the title of the pull request. Use the "Fixes #0000" format.
```
Comment (krlberry) [inline CONTRIBUTING.md]: Expand on the "Fixes #0000" format description. ("#0000 is the ticket number in Redmine)

File: CONTRIBUTING.md (hunk 4 above)
Comment (krlberry) [reply CONTRIBUTING.md:23]: Resolved!

**Review by scsides — APPROVED** [review]

**Pull request merged**